pkg/wire/payload/transaction/base.go

```diff
@@ -199,17 +196,7 @@ func (b *Base) Bytes() []byte {
 	return buf.Bytes()
 }
 
-// UTXOs returns the outputs in the tx
-func (b *Base) UTXOs() []*Output {
-	return b.Outputs
-}
-
-// TXOs returns the inputs in the tx
-func (b *Base) TXOs() []*Input {
-	return b.Inputs
-}
-
-// Witness returns the witnesses in the tx
-func (b *Base) Witness() []*Witness {
-	return b.Witnesses
+// BaseTx retuns a pointer to himself
```

Comment:
One last nit: "returns" . Maybe "BaseTx returns the Base object in a transaction" As we can see it is a pointer. 

```
BaseTx returns the Base object in a transaction
func (b *Base) BaseTx() *Base {
	return b
}
```

Apart from that LGTM